GooglePlayInstant/Editor/PostBuildCommandLineDialog.cs

```diff
@@ -20,9 +20,9 @@ namespace GooglePlayInstant.Editor
     /// <summary>
     /// A CommandLineDialog that waits to run the specified command until after an AppDomain reset.
     ///
-    /// Several seconds after a call to BuildPlayer finishes, Unity reloads all Editor scripts and resets the
-    /// AppDomain, causing any running threads to be aborted. This EditorWindow waits to run the specified
-    /// command until after the AppDomain is reset.
+    /// On Unity versions older than 2018.3, Several seconds after a call to BuildPlayer finishes, Unity reloads all
+    /// Editor scripts and resets the AppDomain, causing any running threads to be aborted. This EditorWindow waits to
+    /// run the specified command until after the AppDomain is reset.
     /// </summary>
```

Comment:
That one sentence now has 3 commas. A few suggestions (note one new word in first line):

A CommandLineDialog that potentially waits to run the specified command until after an AppDomain reset.

Unity versions prior to 2018.3 reload all Editor scripts a few seconds after a call to BuildPlayer finishes. This reload also resets the AppDomain and aborts any actively running threads, including any threads managing a child process. This EditorWindow therefore waits to run the specified command until after the AppDomain is reset.

Reply:
That's much better. Thanks!